jdaviz/core/helpers.py

```diff
@@ -131,8 +131,17 @@ def plugins(self):
         plugins : dict
             dict of plugin objects
         """
-        return {item['label']: widget_serialization['from_json'](item['widget'], None).user_api
-                for item in self.app.state.tray_items}
+        plugins = {item['label']: widget_serialization['from_json'](item['widget'], None).user_api
+                   for item in self.app.state.tray_items}
+
+        # handle renamed plugins during deprecation
+        if 'Orientation' in plugins.keys():
```

Comment:
`.key()` is redundant.

```suggestion
        if 'Orientation' in plugins:
```